roles/dhcp-server/templates/boxes.conf.j2

```diff
@@ -0,0 +1,20 @@
+# /etc/dnsmasq.d/boxes.conf
+#
+
+# defaults:
+# drive
+# tasks
+
+dhcp-option-force=209,"partman-auto/disk=/dev/{{default_install_dev}} grub-installer/bootdev=/dev/{{default_install_dev}} tasks={{default_install_tasks}}"
+
+{% for box in boxes %}
+# {{box.hostname}}
+dhcp-host={{box.mac}},set:{{box.hostname}},{{box.hostname}}{% if box.ip is defined %},{{box.ip}}{% endif %}
+{% if box.ip is defined %}host-record={{box.hostname}},{{box.ip}}{% endif %}
+
+{% if box.disk is defined %}dhcp-option-force=tag:{{box.hostname}},209,"partman-auto/disk=/dev/{{box.disk}} grub-installer/bootdev=/dev/{{box.disk}} tasks={{box.tasks|default()}}" {% endif %}
+
+{% if box.comment is defined %}# {{box.comment}}{% endif %}
```

Comment:
Seems like you'd probably want the comment earlier?

Reply:
fixed.  
{{box.hostname}} {{box.comment}} so that the hostname before the comment 
if no comment, then  this is the first line: dhcp-host={{box.mac}},set:{{box.hostname}}...